From: paul <paul@claws-mail.org>
Date: Mon, 12 Jul 2021 09:20:21 +0000 (+0100)
Subject: fix bug 4496 'Fancy crashes when opening an email'
X-Git-Url: http://git.claws-mail.org/?p=claws.git;a=commitdiff_plain;h=f459d1c27426a10af90f8ef73aef78ec846ebffb;hp=9c2585c58b49815a0eab8d683f0a94f75cbbe64e

fix bug 4496 'Fancy crashes when opening an email'

Patch by Jonathan Boeing
---

diff --git a/src/plugins/fancy/fancy_viewer.c b/src/plugins/fancy/fancy_viewer.c
index 6ff5f1244..d57d33aa7 100644
--- a/src/plugins/fancy/fancy_viewer.c
+++ b/src/plugins/fancy/fancy_viewer.c
@@ -96,7 +96,7 @@ static void fancy_apply_prefs(FancyViewer *viewer)
 #endif
 		NULL);
 	if (fancy_prefs.stylesheet == NULL || strlen(fancy_prefs.stylesheet) == 0) {
-		gchar **msg_font_params = g_strsplit(prefs_common.textfont, " ", 0);
+		gchar **msg_font_params = g_strsplit(prefs_common_get_prefs()->textfont, " ", 0);
 		guint params_len = g_strv_length(msg_font_params);
 
 		if (params_len > 0) {
